crypto/mbedtls/Kconfig

```diff
@@ -0,0 +1,80 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig CRYPTO_MBEDTLS
+	bool "Mbed TLS Cryptography Library"
+	default n
+	---help---
+		Enable support for Mbed TLS.
+
+if CRYPTO_MBEDTLS
+
+config MBEDTLS_VERSION
+	string "MBEDTLS Version"
+	default "2.25.0"
+
+menuconfig MBEDTLS_APPS
```

Comment:
normally, we reuse CRYPTO_MBEDTLS for module by changing it to tristate and update the if condition in Make.defs:
```
ifneq ($(CONFIG_CRYPTO_MBEDTLS),)
CONFIGURED_APPS += $(APPDIR)/crypto/mbedtls
endif
```

Reply:
This seems wrong. The applications and the library are two separate things. Enabling the library should have no impact (besides dependency) on the applications.

Reply:
Yes, in this case(not zmodem), it may make sense to add a new variable for module.